include/itemlistformaction.h

```diff
@@ -92,6 +92,13 @@ class ItemListFormAction : public ListFormAction {
 		const std::string& url,
 		FmtStrFormatter& fmt);
 
+	std::vector<ItemPtrPosPair> visible_items;
+	int old_itempos;
+	std::shared_ptr<RssFeed> feed;
+	std::string search_phrase;
+	bool apply_filter;
+	Matcher matcher;
```

Comment:
I think these fields can be made `private` because they don't seem to be used in the derived class.